src/main/java/uk/gov/justice/digital/hmpps/keyworker/config/SentryConfig.kt

```diff
@@ -20,14 +20,21 @@ class SentryConfig {
       context.customSamplingContext?.let {
         val request = it["request"] as HttpServletRequest
         when (request.method) {
-          "GET" if (matches("/key-worker/offender/[A-Z][0-9]{4}[A-Z]{2}", request.requestURI)) -> {
-            0.0025
+          "GET" if (request.requestURI.isHighUsage()) -> {
+            0.001
           }
 
           else -> {
-            0.05
+            0.02
           }
         }
       }
     }
+
+  private fun String.isNoSample(): Boolean =
+    this.startsWith("GET /health") or this.startsWith("GET /info") or this.startsWith("GET /swagger-ui")
+
+  private fun String.isHighUsage(): Boolean =
+    matches("/key-worker/offender/[A-Z][0-9]{4}[A-Z]{2}", this) or
+      matches("/prisons/[A-Z]{3}/prisoners/[A-Z][0-9]{4}[A-Z]{2}/keyworkers/current", this)
```

Comment:
Regex should match the GET /prisons/{prisonCode}/prisoners/{prisonNumber}/keyworkers/current endpoint